unit/goto-programs/goto_program_validate.cpp

```diff
@@ -0,0 +1,429 @@
+/*******************************************************************\
+
+ Module: Unit tests for goto program validation
+
+ Author: Diffblue Ltd.
+
+ Date: Oct 2018
+
+\*******************************************************************/
+
+/// \file
+/// Unit tests for goto program validation
+
+#include <testing-utils/catch.hpp>
+#include <testing-utils/message.h>
+
+#include <goto-programs/goto_convert_functions.h>
+#include <goto-programs/validate_goto_model.h>
+
+#include <util/arith_tools.h>
+#include <util/c_types.h>
+#include <util/std_code.h>
+#include <util/validate.h>
+
+namespace
+{
+code_function_callt make_void_call(const symbol_exprt &function)
+{
+  code_function_callt ret;
+  ret.function() = function;
```

Comment:
There is a constructor in `code_function_callt` for this case.

Reply:
Done.